src/pages/favorites.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect } from 'react';
+import PageTitle from '../atoms/pageTitle';
+import Cardlist from '../molecules/cardlist';
+import store from '../Redux/store';
+
+const Favorites = () => {
+  // eslint-disable-next-line
+    useEffect (()=> {window.scrollTo(0,0)},[]);
+
+  return (store.getState().favorites.length === 0) ? (
```

Comment:
For big if blocks like this consider using react control statements https://www.npmjs.com/package/jsx-control-statements